aerofiles/seeyou/reader.py

```diff
@@ -68,6 +81,23 @@ def decode_waypoint(self, fields):
             'description': self.decode_description(fields[10]),
         }
 
+    def decode_task_info(self, tasks, fields):
+        if fields[0] == 'Options':
+            tasks[-1]['Options'] = self.decode_task_options(fields)
+        elif fields[0].startswith('ObsZone'):
+            tasks[-1]['ObsZones'].append(self.decode_task_ObsZone(fields))
+        else:
```

Comment:
does that not imply a complete removal of `decode_task_info()`?
decoding the simple task lines is done by `decode_task_options()` and `decode_task_obs_zone()`.